controlsfx/src/main/java/org/controlsfx/control/CheckListView.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2013, 2018 ControlsFX
+ * Copyright (c) 2013, 2018, 2019 ControlsFX

```

Comment:
This should be Copyright (c) 2013, 2019 ControlsFX

Reply:
Apologies, I mis-interpreted the contributing guide. I've pushed a second commit with the requested change